app/controllers/api/v1/people_controller.rb

```diff
@@ -5,7 +5,7 @@ module V1
     class PeopleController < ApiController
       def index
         people = People::Finder.new(filter_params).call
-        paginate people, include: PersonSerializer::INCLUDED_DETAIL
+        paginate Kaminari.paginate_array(people.map(&:latest_profile)), include: ProfileSerializer::INCLUDED_DETAIL
```

Comment:
I noticed 
`latest_profile` is a method in the model`Person`
You might want to use that instead of `people.map(&:latest_profile`